generic/tests/netperf.py

```diff
@@ -93,6 +93,7 @@ def env_setup(session, ip, user, port, password):
         error.context("Setup env for %s" % ip)
         ssh_cmd(session, "iptables -F", ignore_status=True)
         ssh_cmd(session, "service iptables stop", ignore_status=True)
+        ssh_cmd(session, "systemctl stop firewalld.service", ignore_status=True)
```

Comment:
systemctl is used for newer guest like rhel7.4. the service is used for older guest like rhel6.